README.md

```diff
@@ -88,6 +80,25 @@ Create a `appsettings.json` file in the AELFFaucet.Web directory of the project
   }
 }
 ```
+
+## Setup secrets.json file
+
+Run this following command in terminal of the AELFFaucet.Web directory to generate the secrets.json file with all ENV keys: 
+
+```base title="Terminal"
+dotnet user-secrets init
+dotnet user-secrets set "ApiConfig:BaseUrl" "http://3.25.10.185:8000"
+dotnet user-secrets set "ApiConfig:PrivateKey" "b192e307da4c4e1a00eeed442a4b5e8e0b7b2c6f838d0472627b846e5280c51c"
+dotnet user-secrets set "ApiConfig:NftSeedPrivateKey" "97a1747af1539a6b839c92227c8f5ca90ef0e7de70e5c2f46c64f27b12fabdd7"
+dotnet user-secrets set "ApiConfig:BaseUrlForMainchain" "https://aelf-test-node.aelf.io"
+dotnet user-secrets set "ApiConfig:BaseUrlForSidechain" "https://tdvw-test-node.aelf.io"
+dotnet user-secrets set "ApiConfig:SendCount" "100"
+dotnet user-secrets set "ApiConfig:RecaptchaVerifyUrl" "https://www.google.com/recaptcha/api/siteverify"
```

Comment:
This Url don't need to be in secrets.json. This should be in appsettings. Same for ApiConfig:SendCount, ApiConfig:BaseUrlForSidechain, ApiConfig:BaseUrlForMainchain and ApiConfig:BaseUrl. Only put keys into secrets.json. Non-sensitive information should go to appsettings.json